test.ts

```diff
@@ -282,7 +297,28 @@ Deno.test({
                   },
                 },
               },
-              "assertionType": "json",
+              "imports": [
+                {
+                  "assertions": {
+                    "Known": {
```

Comment:
Yeah, I guess we can switch to blessed fields for known assertions on the serialization layer. Just now wondering if we should store it like that in the first place..

Reply:
If everything still works then storing it like that might be better.

Reply:
I've been mulling over schemas that:
1. Transfer well to dynamic imports with potential unknowns
2. Don't then add a ton of boilerplate for static imports
3. Don't conflate dynamic/unknown assertions with unspecified ones

And landed on:
```ts
interface Import {
  assertions: "unknown" | Record<string, string>,
}
```
where for static imports you can just pretend the `"unknown" |` isn't there. This would use the `maybe_static()` method from https://github.com/denoland/deno_graph/pull/228#discussion_r1103861394 and just mark dynamic imports as `"unknown"` if there is _any_ dynamic part of it. We have to give up the blessed fields like `assertionType?: string` since it would violate either (1) or (3) but `assertions.type` is pretty much the same.

Reply:
Sounds good.